packages/netlify/package.json

```diff
@@ -30,10 +30,10 @@
   ],
   "scripts": {
     "build": "tsc",
-    "test-fn": "mocha --exit --timeout 20000 --file \"./test/setup.js\" test/functions/",
-    "test-static": "mocha --exit --timeout 20000 --file \"./test/setup.js\" test/static/",
-    "test": "pnpm test-fn && pnpm test-static",
-    "test:hosted": "mocha --exit --timeout 30000 test/hosted"
+    "test-fn": "astro-scripts --test ./test/functions/*.test.js",
+    "test-static": "astro-scripts --test ./test/static/*.test.js",
+    "test": "pnpm run test-fn && pnpm run test-static",
+    "test:hosted": "astro-scripts --test ./test/hosted/*.test.js"
```

Comment:
```suggestion
    "test:hosted": "astro-scripts test ./test/hosted/*.test.js"
```

Reply:
Thank you 🙇 I added it to https://github.com/withastro/adapters/pull/158/commits/f26386e93078232cf992f934a78908549b10c537 but I see you also pushed it 😂